src/tequila/simulators/simulator_aqt.py

```diff
@@ -0,0 +1,86 @@
+from calendar import c
+from tequila.utils.keymap import KeyMapRegisterToSubregister
+from tequila import BitString 
+from tequila.simulators.simulator_qiskit import BackendCircuitQiskit, BackendExpectationValueQiskit, TequilaQiskitException
+from qiskit_aqt_provider import AQTProvider
+from qiskit_aqt_provider.aqt_resource import AQTResource
+from qiskit.circuit import QuantumCircuit
+from tequila.wavefunction.qubit_wavefunction import QubitWaveFunction
+from tequila import TequilaException
+import qiskit
+from typing import Union
+
+class TequilaAQTException(TequilaQiskitException):
+    def __str__(self):
+        return "Error in AQT backend:" + self.message
+
+
+class BackendCircuitAQT(BackendCircuitQiskit):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.device = self.get_backend()
```

Comment:
*args/**kwargs should be passed (otherwise no token can be set from the outside)